adlfs/multithread.py

```diff
@@ -22,8 +22,6 @@
 from .transfer import ADLTransferClient
 from .utils import commonprefix, datadir, read_block, tokenize
 
-MAXRETRIES = 5
```

Comment:
Yes, we catch any exception, print out a debug message, and retry for some configurable number of times. If the max is reached, then we throw a RuntimeError.